qutebrowser/mainwindow/prompt.py

```diff
@@ -422,6 +422,17 @@ def prompt_item_focus(self, which):
         except UnsupportedOperationError:
             pass
 
+    @cmdutils.register(instance='prompt-container', scope='window',
+                       modes=[usertypes.KeyMode.prompt])
+    def prompt_yank(self):
+        """Yank URLs or other data in prompts."""
+        question = self._prompt.question
+        s = None
+        if question and hasattr(question, 'yank_text'):
```

Comment:
Various things to note here:

- Why would `question` ever be falsey (or `None`) here, i.e. why the `if question`?
- Why would `question` ever not have that attribute? You set it in `Question.__init__`, so that check does nothing at all (and generally, `hasattr()` should be avoided if possible).
- Why do you want to yank `None` (what does that do?) rather than e.g. show an error if there's nothing to yank?